samtranslator/internal/schema_source/aws_serverless_api.py

```diff
@@ -100,6 +100,7 @@ class UsagePlan(BaseModel):
 
 class Auth(BaseModel):
     AddDefaultAuthorizerToCorsPreflight: Optional[bool] = auth("AddDefaultAuthorizerToCorsPreflight")
+    AddApiKeyRequiredToCorsPreflight: Optional[bool]  # TODO Add Docs
```

Comment:
Should also add this property to `Globals` schema.

Reply:
Will do!